README.md

```diff
@@ -8,11 +8,14 @@ This repo provides packages for easily working with passwords in Haskell.  This
 is mainly to be used for web applications that need to receive plain-text
 passwords from users, and store hashed passwords in a database.
 
-The [password](./password) package provides datatypes and functions for working
+The [password-types](./password-types) package provides datatypes for working
 with plain-text and hashed passwords.
```

Comment:
```suggestion
The [password-types](./password-types) package provides canonical datatypes for
plain-text and hashed passwords.  This package has minimal dependencies.
It is meant to be used as a base for any other packages that want `Password` data types.
```

I don't feel super strongly that this should necessarily go into the README, but I'd like to have the idea that `password-types` has minimal dependencies and provides canonical data types included in some documentation somewhere.  Ideally other packages that provide/operate on a `Password` type would instead take a dependency on `password-types` so that these packages could interop.